liri.js

```diff
@@ -0,0 +1,161 @@
+require("dotenv").config();
+
+var keys = require("./keys.js");
+
+var axios = require("axios");
+
+var moment = require("moment");
+
+var fs = require('fs');
+
+var Spotify = require('node-spotify-api');
+
+var spotify = new Spotify(keys.spotify);
+
+moment().format("MM/DD/YYYY");
+ 
+
+
+
+//////////   Switch Statement  /////////////////////
+var pick = function(action, value){
+switch (action) {
+case "concert-this":
+    concertSearch(value);
+    break;
+
+case "spotify-this-song":
+    spotifySearch(value);
+    break;
+
+case "movie-this":
+    movieSearch(value);
+    break;
+
+case "do-what-it-says":
+    textSearch();
+    break;
+}}
+
+
+////////// Identify process.argv variables ////////////////////
+
+var action = process.argv[2];
+var value = process.argv[3];
+
+
+//////// SPOTIFY SHIT BELOW  ////////////////////////////////////
+////// To Do / Having Trouble
+// 
+// 
+// log default for empty input
+///////////////////////////////////////////
+var showArtist = function (artist) {
+    return artist.name;
+};
+
+var spotifySearch  = function(songName) {
+spotify.search({ type: 'track', query: songName }, function(err, data) {
+    if (err) {
+      return console.log('Error occurred: ' + err);
+    }
+    else if (process.argv[3] === null){
+        songName === 'the sign'
+    }
+
+    var songs = data.tracks.items;
+
+    for (var i=0; i < songs.length; i++){
+        console.log('-----------------------------------------------');
+        // console.log(i);
+        // console.log(data.tracks.items[0].artists);
+        console.log('artist(s): ' + songs[i].artists.map(showArtist));
+        console.log('song name: ' + songs[i].name);
+        console.log('preview song: ' + songs[i].preview_url);
+        console.log('album: ' + songs[i].album.name);
+        console.log('-----------------------------------------------');
+
+        }
+  });
+}
+
+//////////////////////////////////////////////
+//////// BANDSINTOWN BELOW //////////////////////////////////
+///// To Do / Having Trouble
+//
+// use moment to format date   ?????
+//
+//
+///////////////////////////////////////////
+function concertSearch(){
```

Comment:
You would also have to make sure you are passing down a value like `function concertSearch(value)` to have it pass down to the below line